_faqs/cpu-architectures.md

```diff
@@ -0,0 +1,5 @@
+---
+question: What CPU architectures are supported?
+---
+
+X86_64 (Intel) and AARCH64 (Arm) are supported. Mixed architectures (e.g. some hosts x86_64, some - aarch64) are not supported yet.
```

Comment:
```suggestion
Both x86_64 (Intel) and AARCH64 (Arm) are supported. Mixed architectures (e.g. some hosts x86_64, some - aarch64) are not supported yet.
```

Can someone elaborate on why mixed architectures are not supported? What is needed to support this?